tests/support_server/configure.pm

```diff
@@ -47,7 +47,7 @@ sub _remove_installation_media_and_add_network_repos {
     if (get_var("SLENKINS_REPO")) {
         $script .= "zypper -n --no-gpg-checks ar --refresh '" . get_var("SLENKINS_REPO") . "' slenkins\n";
     }
-
+    $script .= "zypper --gpg-auto-import-keys ref -f\n";
```

Comment:
Do we really need force flag here? Other than that LGTM.

Reply:
propably not. but i dont think is destructive in some way. i might be wrong

Reply:
Sure, we can leave it. In case of repo refresh it should not change anything.